.gitignore

```diff
@@ -1,2 +1,3 @@
-calcite-tmp/
-vendor/
\ No newline at end of file
+/calcite-tmp/
```

Comment:
I think we can remove the `/calcite-tmp/` directory from the .gitignore because we now use the `avatica-tmp` directory.